src/backend/distributed/transaction/remote_transaction.c

```diff
@@ -65,6 +69,134 @@ static char *IsolationLevelName[] = {
 };
 
 
+PG_FUNCTION_INFO_V1(citus_internal_start_management_transaction);
+PG_FUNCTION_INFO_V1(execute_command_on_other_nodes);
+PG_FUNCTION_INFO_V1(citus_mark_object_distributed);
+PG_FUNCTION_INFO_V1(commit_management_command_2PC);
+
+bool IsMainDBCommand = false;
+FullTransactionId OuterXid;
+static MultiConnection *MainDBConnection = NULL;
+bool IsMainDB = true;
+char *MainDb = "";
```

Comment:
nit: Should we move UDF & variable declarations to top of the file?

It'd be useful to add a comment for MainDBConnection too.

Reply:
Moved and added comments for all of them.